process/datadog_checks/process/process.py

```diff
@@ -199,21 +200,23 @@ def psutil_wrapper(self, process, method, accessors, try_sudo, *args, **kwargs):
                     try:
                         result[acc] = getattr(res, acc)
                     except AttributeError:
-                        self.log.debug("psutil.%s().%s attribute does not exist", method, acc)
+                        self.log.debug("psutil.{0}().{1} attribute does",\
+                                        "not exist".format(method, acc))
         except (NotImplementedError, AttributeError):
-            self.log.debug("psutil method %s not implemented", method)
+            self.log.debug("psutil method {} not implemented".format(method))
         except psutil.AccessDenied:
-            self.log.debug("psutil was denied acccess for method %s", method)
+            self.log.debug("psutil was denied access for method {}".format(method))
             if method == 'num_fds' and Platform.is_unix() and try_sudo:
                 try:
                     # It is up the agent's packager to grant corresponding sudo policy on unix platforms
                     process_ls = subprocess.check_output(['sudo', 'ls', '/proc/{}/fd/'.format(process.pid)])
                     result = len(process_ls.splitlines())
                     
                 except subprocess.CalledProcessError as e:
-                    self.log.exception("trying to retrieve %s with sudo failed with return code %d", method, e.returncode)
+                    self.log.exception("trying to retrieve {0} with sudo failed with ",\
+                                       "return code {1}".format(method, e.returncode))
```

Comment:
I think we don't need to use indexed string placements 👍 

As well, since the "s are at the same indentation, you don't need ``, \``:
```
self.log.exception("trying to retrieve {} with sudo failed with "
                   "return code {}".format(method, e.returncode))